modules/node/init.zsh

```diff
@@ -14,6 +14,15 @@ if [[ -s "$HOME/.nvm/nvm.sh" ]]; then
 elif (( $+commands[brew] )) && [[ -d "$(brew --prefix nvm 2>/dev/null)" ]]; then
   source $(brew --prefix nvm)/nvm.sh
 
+# Load manually installed nodenv into the shell session.
+elif [[ -s "$HOME/.nodenv/bin/nodenv" ]]; then
+  path=("$HOME/.nodenv/bin" $path)
```

Comment:
This is not really nice. I am about to merge your changes to my fork of prezto, but in order to do this I would like to have a nicer solution e.g. at least putting `$HOME/.nodenv/bin` into the .zprofile. hmmm